app/decorators/module_overview_decorator.rb

```diff
@@ -93,7 +93,7 @@ def heading(page)
   # @param pages [Array<Training::Page, Training::Question, Training::Video>]
   # @return [String]
   def page_count(pages)
-    "(#{pages.size} pages)" unless pages.one?
+    I18n.t('my_learning.page_count', num: pages.size) unless pages.one?
```

Comment:
Does this exist?

Reply:
This is currently in staging in the CMS and I have updated the `en.yml` in the latest commit to include this

Reply:
locales count spec next